### PR TITLE
feat: add integration test cicd

### PR DESCRIPTION
**Pull request by wuyou10206**
add integration test cicd

### CODE REVIEW

File: .github/workflows/integration-test-pkg.yml [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+jobs:
+  java-sdk-test-standalone-0:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/4paradigm/hybridsql:0.4.0
+    env:
+      OS: linux
+    steps:
+      - uses: actions/checkout@v1
```
Comment (aceforeverd) [inline .github/workflows/integration-test-pkg.yml]: use `checkout@v2`

File: .github/workflows/integration-test-pkg.yml [hunk 2]
```diff
@@ -0,0 +1,197 @@
+name: INTEGRATION-TEST-PKG
+
+on:
+  pull_request:
```
Comment (aceforeverd) [inline .github/workflows/integration-test-pkg.yml]: if test is unstable, maybe rm `pull_request` trigger

File: cases/function/function/test_date.yaml [hunk 3]
```diff
@@ -140,4 +140,4 @@ cases:
       - ["{0}.c1","{0}.c2","{0}.c3","{0}.c5","{0}.c6","{0}.c8","{0}.c9"]
     sql: select id as id,d[0](d[1]) as e1 from {0};
     expect:
-      success: false
+      success: false
```
Comment (vagetablechicken) [inline cases/function/function/test_date.yaml:143]: newline

File: test/steps/deploy_fedb.sh [hunk 4]
```diff
@@ -112,4 +115,4 @@ function deploy_api_server() {
   cd ..
   sleep 5
 }
-deploy_api_server fedb-apiserver-1
\ No newline at end of file
+deploy_api_server openmldb-apiserver-1
```
Comment (vagetablechicken) [inline test/steps/deploy_fedb.sh:118]: newline

File: test/steps/modify_apiserver_config.sh [hunk 5]
```diff
@@ -23,16 +23,16 @@ FEDB_SERVER_VERSION=$5
...
-    sed -i "s#<parameter name=\"fedbPath\" value=\".*\"/>#<parameter name=\"fedbPath\" value=\"${ROOT_DIR}/OpenMLDB/build/bin/fedb\"/>#" test_suite/"${CASE_XML}"
+    sed -i "s#<parameter name=\"fedbPath\" value=\".*\"/>#<parameter name=\"fedbPath\" value=\"${ROOT_DIR}/build/bin/openmldb\"/>#" test_suite/"${CASE_XML}"
 fi
 # modify pom
 cd "${ROOT_DIR}" || exit
-cd java/hybridsql-test/fedb-sdk-test || exit
+cd test/integration-test/openmldb-test-java/openmldb-sdk-test || exit
 sed -i "s#<openmldb.version>.*</openmldb.version>#<openmldb.version>${FEDB_SDK_VERSION}</openmldb.version>#" pom.xml
 
 cd "${ROOT_DIR}" || exit
```
Comment (vagetablechicken) [inline test/steps/modify_apiserver_config.sh:38]: newline

File: test/steps/openmldb-batch-test.sh [hunk 6]
```diff
@@ -16,9 +16,14 @@
...
+cd test/test-tool/command-tool || exit
 mvn clean install -Dmaven.test.skip=true
-cd "${ROOT_DIR}"/java/hybridsql-test/sparkfe_test/ || exit
+cd "${ROOT_DIR}" || exit
+
+cd "${ROOT_DIR}"/test/integration-test/openmldb-test-java || exit
+mvn clean install -Dmaven.test.skip=true
+
+cd "${ROOT_DIR}"/test/batch-test/openmldb-batch-test/ || exit
 mvn clean test
```
Comment (vagetablechicken) [inline test/steps/openmldb-batch-test.sh:29]: n

File: test/steps/fedb_sdk_test.properties [hunk 7]
```diff
@@ -1,5 +1,5 @@
 FEDB_SDK_VERSION=0.2.3
 FEDB_SERVER_VERSION=0.2.3
-FEDB_PY_SDK_VERSION=2.3.0
+FEDB_PY_SDK_VERSION=0.2.3
```
Comment (vagetablechicken) [inline test/steps/fedb_sdk_test.properties:3]: There're so many files renamed fedb to openmldb, we need rename here?

**Review by vagetablechicken — APPROVED** [review]

File: cases/function/function/test_date.yaml (hunk 3 above)
Comment (vagetablechicken) [reply cases/function/function/test_date.yaml:143]: later

File: test/steps/fedb_sdk_test.properties (hunk 7 above)
Comment (vagetablechicken) [reply test/steps/fedb_sdk_test.properties:3]: later

**Review by dl239 — APPROVED** [review]

**Pull request merged**